bitnami/odoo/Chart.yaml

```diff
@@ -34,4 +34,4 @@ maintainers:
 name: odoo
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/odoo
-version: 24.0.0
```

Comment:
Could you please bump the chart version again?

version 24.0.1 has already been released: https://github.com/bitnami/charts/blob/main/bitnami/odoo/Chart.yaml

Reply:
Done, thanks for pointing out.